comfy/model_management.py

```diff
@@ -334,11 +334,13 @@ def amd_min_version(device=None, min_rdna_version=0):
 AMD_RDNA2_AND_OLDER_ARCH = ["gfx1030", "gfx1031", "gfx1010", "gfx1011", "gfx1012", "gfx906", "gfx900", "gfx803"]
 
 try:
-    if is_amd():
```

Comment:
I think we still need the `is_amd()` check here, the following nested logic applies only to amd cards.